packages/site/src/layout/NavMenu.tsx

```diff
@@ -137,11 +146,21 @@ export const NavMenu = ({ mainContentRef }: NavMenuProps) => {
 export const StyledLink = ({
   to,
   children,
-}: PropsWithChildren<{ readonly to: string }>) => {
+  selectedRef,
+}: PropsWithChildren<{
+  readonly to: string;
+  readonly selectedRef: React.RefObject<HTMLAnchorElement>;
+}>) => {
+  const isSelected =
+    location.pathname === to || (location.pathname === "/" && to === "/?new");
```

Comment:
Needed to add the `to === "/?new"` because Home isn't in a disclosure, but we still need it to be highlighted when selected/the URL matches

Reply:
I think this is a great time to use a constant for this this value and maybe even a comment about it. Can we grab this value from another file so that when we hard launch we don't have to come back and update here?

Reply:
To sum up a few conversations surrounding this:
Although I understand how this is a good idea in general, we're actually just going to be getting rid of that code entirely, probably in January. I've added an explanatory/TODO comment above this logic here 0cc72051928439d3de0a6cf7dbcb5152fb1ad2bb